### PR TITLE
Tell user to say no when asked to enable WebAuth

### PR DESCRIPTION
**Pull request by jakeinater**
edited tutorial instructions

### CODE REVIEW

File: docs/docs/tutorial/chapter4/authentication.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Run this setup command to get the internals of dbAuth added to our app:
 yarn rw setup auth dbAuth
 ```
 
-When asked if you want to override the existing file `/api/src/lib/auth.{js,ts}` say yes. The shell `auth.{js,ts}` that's created in a new app makes sure things like the `@requireAuth` directive work, but now we'll replace it with a real implementation.
+When asked if you want to override the existing file `/api/src/lib/auth.{js,ts}` say yes. The shell `auth.{js,ts}` that's created in a new app makes sure things like the `@requireAuth` directive work, but now we'll replace it with a real implementation. Say no when asked to Enable WebAuthn support.
```
Comment (dac09) [inline docs/docs/tutorial/chapter4/authentication.md:134]:
```suggestion
When asked if you want to override the existing file `/api/src/lib/auth.{js,ts}` say yes. The shell `auth.{js,ts}` that's created in a new app makes sure things like the `@requireAuth` directive work, but now we'll replace it with a real implementation. When prompted to "Enable WebAuthn support", pick no - this is a separate piece of functionality we won't need for the tutorial.
```

File: docs/docs/tutorial/chapter4/authentication.md [hunk 2]
```diff
@@ -474,7 +474,7 @@ Yet another generator is here for you, this time one that will create pages for
 yarn rw g dbAuth
 ```
 
-Again several pages will be created and some post-install instructions will describe next steps. But for now, try going to [http://localhost:8910/login](http://localhost:8910/login):
+Say no when asked to Enable WebAuthn support. Again several pages will be created and some post-install instructions will describe next steps. But for now, try going to [http://localhost:8910/login](http://localhost:8910/login):
```
Comment (dac09) [inline docs/docs/tutorial/chapter4/authentication.md:477]: No need to repeat this here again, I think!

**Comment by dac09** [comment]
Hey @jakeinater thanks for the PR. Left a couple of comments.

Would you mind also updating the current version of the docs? You'll find them here: https://github.com/redwoodjs/redwood/tree/main/docs/versioned_docs/version-3.0/tutorial/chapter4

File: docs/docs/tutorial/chapter4/authentication.md (hunk 1 above)
Comment (jakeinater) [reply docs/docs/tutorial/chapter4/authentication.md]: Added!

File: docs/docs/tutorial/chapter4/authentication.md (hunk 2 above)
Comment (jakeinater) [reply docs/docs/tutorial/chapter4/authentication.md]: Sure, deleted.

**Comment by jakeinater** [comment]
Should be good now I think @dac09

**Review by dac09 — APPROVED** [review]

**Pull request merged**